integrasjonstest/src/test/kotlin/no/nav/helsearbeidsgiver/inntektsmelding/integrasjonstest/TilgangskontrollIT.kt

```diff
@@ -71,9 +113,29 @@ class TilgangskontrollIT : EndToEndTest() {
 
     @Test
     fun `forespoersel - skal få melding om at forespørsel ikke finnes`() {
```

Comment:
Denne testen tester vel egentlig bare mocken? Kunne egentlig bare vært kutta ut imho. Navnet blir uansett litt misvisende, om du vil beholde den.

Reply:
Godt poeng. Jeg hadde bare øyne for å få de eksisterende testene til å fungere. Jeg fjerner den.